torch/csrc/distributed/rpc/FutureMessage.cpp

```diff
@@ -0,0 +1,61 @@
+#include <torch/csrc/distributed/rpc/FutureMessage.h>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+void FutureMessage::wait() {
+  std::unique_lock<std::mutex> lock(mutex_);
+  finished_cv_.wait(lock, [this]{return completed_.load();});
+}
+
+void FutureMessage::markCompleted(Message message) {
+  {
+    std::unique_lock<std::mutex> lock(mutex_);
+    TORCH_CHECK(!completed());
```

Comment:
This should probably be `TORCH_ASSERT`

Reply:
Do you mean `AT_ASSERT`? It seems there is no `TORCH_ASSERT` yet.